app/assets/stylesheets/alchemy/base.scss

```diff
@@ -541,6 +541,10 @@ div.browse {
   width: 100%;
 }
 
+.hidden {
+  display: none;
+}
```

Comment:
I don't like class names that are verbs. A class name should describe the state not the action it does. As a long as no animations are involved for sure.